src/quality_result_gui/quality_error_manager.py

```diff
@@ -62,6 +64,8 @@ class QualityResultManager(QObject):
     error_checked = pyqtSignal(str, bool)
     quality_error_selected = pyqtSignal(QualityError, SelectionType)
 
+    SHORTCUT_TOGGLE_ERRORS_ON_MAP_FILTER = "Ctrl+Alt+L"
```

Comment:
I think that `L` stands for Laatu. Should we have the default shortcut in this public repo something more. And that is also reserved shortcut by default. We could change the shortcut to our own in our private implementation.

Reply:
Changed to Alt+Q (as in quality)